metabase/metabase
# Allow decimal settings values

Closes https://github.com/metabase/metabase/issues/24657

## Changes

Allows decimal numeric settings values and hides native numeric input controls.

## How to verify

- Admin settings -> Caching
- Try to change `MINIMUM QUERY DURATION` to a decimal value and ensure it works
